libretroshare/src/rsitems/rsserviceids.h

```diff
@@ -40,7 +40,6 @@ enum class RsServiceType : uint16_t
 	FILE_DATABASE              = 0x0019,
 	SERVICEINFO                = 0x0020,
 	BANDWIDTH_CONTROL          = 0x0021,
-	MAIL                       = 0x0022,
```

Comment:
If possible keep the line with "unused since XXX (formerly MAIL)". So when we use this number, we know that we dont have risks to collide with an existing service id that was only recently removed.